minify.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
```

Comment:
I like the JS approach, but I'm wondering if we could make use of the [clean-css-cli](https://www.npmjs.com/package/clean-css-cli) package instead, so we have less to maintain?

Reply:
Yeah, instead of reading the files with `fs` we could make use of the clean-css-cli package which would completely eliminate the need for any JavaScript to be added as `minify.js` wouldn't be needed anymore. It would be just 1 dependency using `clean-css-cli` and 2 with `fs` + `clean-css`. I'm happy to use clean-css-cli if you are! It definitely will be "slimmer" (since its all CLI commands) than the JS approach (20-30 lines) I spun up.

Reply:
I agree, it'd be nice to have less to maintain and I'm subconsciously trying to avoid using JS in this project 😅